packages/docs/src/styles/_examples.scss

```diff
@@ -213,6 +213,12 @@ $options-margin: $pt-grid-size * 3;
   animation: docs-wiggle-rotate $pt-transition-duration $pt-transition-ease infinite;
 }
 
+.docs-loading-skeleton-example-box {
+  flex: 1;
+  margin-right: $pt-grid-size * 1.5;
+  height: $pt-grid-size * 15;
```

Comment:
if the height isn't fixed, the card size will change as it transitions between the loading / non-loading states